src/Amadeus/Client/Struct/Service/StandaloneCatalogue/FareInfo.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
If you have copied an existing class, please just use the existing class to avoid duplicate code.

Reply:
We need FlightInfo.php here because of request parameter is different from one used in informative pricing request. We need instead of used in InformativePricing Request.

Reply:
You can just use the existing class? Why do you need to make a copy of it?

Reply:
As suggested, Now I make subClass of existing class and used same method from extended class.